src/components/Post/ActionsBar/Reactionbar/ReactionButton.tsx

```diff
@@ -129,10 +133,13 @@ const ReactionButton: FC<IReactionButtonProps> = ({
 			setReactions(newReactions);
 			const actionName = `${reacted ? 'remove' : 'add'}${commentId ? 'CommentOrReply' : 'Post'}Reaction`;
 			const { data, error } = await nextApiClientFetch<MessageType>(`api/v1/auth/actions/${actionName}`, {
+				commentAuthorId: commentAuthorId || null,
```

Comment:
please verify once but i don't think we need to send this data anymore, since we fetch it from the BE anyway